plugin-build/src/test/kotlin/io/sentry/android/gradle/BaseSentryPluginTest.kt

```diff
@@ -99,10 +99,14 @@ abstract class BaseSentryPluginTest(
             arrayOf("7.0.4", "7.2"),
             arrayOf("7.1.2", "7.2"),
             arrayOf("7.1.2", "7.3.3"),
-            arrayOf("7.1.3", "7.4"),
-            arrayOf("7.2.0", "7.4"),
-            arrayOf("7.3.0-beta01", "7.4"),
-            arrayOf("7.4.0-alpha01", "7.4.2")
+            arrayOf("7.1.3", "7.2"),
+            arrayOf("7.1.3", "7.5"),
+            arrayOf("7.2.1", "7.3.3"),
+            arrayOf("7.2.1", "7.5"),
+            arrayOf("7.3.0-beta05", "7.3.3"),
+            arrayOf("7.3.0-beta05", "7.5"),
+            arrayOf("7.4.0-alpha08", "7.3.3."),
+            arrayOf("7.4.0-alpha08", "7.5")
```

Comment:
Alphas we can test only the latest Gradle version, so here in both cases would be `7.3.0-beta05` and 7.5, `7.4.0-alpha08` and 7.5

Reply:
updated